distributed/protocol/tests/test_protocol.py

```diff
@@ -121,6 +121,27 @@ def test_maybe_compress(lib, compression):
         assert compressions[rc]["decompress"](rd) == payload
 
 
+@pytest.mark.parametrize(
```

Comment:
This currently breaks on `main` due to the way the `compression` default value is implemented on `maybe_compress`.

Reply:
It seems reasonable to me to make this behavior work as you have it here.